### ISSUE TITLE
Crash on the preview of running tasks

### ISSUE THREAD

**Issue by hpsaturn** [lunokjod/watch#1]
## Overview

After open two apps and enter to this section, after some time we have:

The device only has WiFi provisioning, not BLE enable.

``` 
[528096451][event] UI: Touchpad long tap
[UI] LunokIoTApplication: 0x3ffb3094 begin
[UI] Application: 0x3ffb3094 'Task switcher' goes to front
[UI] KillApplicationTask: 0x3ffb5714 'Analogic watchface' closing...
[0x3ffb5714]['Analogic watchface'] Watchface is gone
[UI] LunokIoTApplication: 0x3ffb5714 ends
[528543421][lunokIoT] SQL: Time taken: 71842
[528547045][lunokIoT] ESP32: Free stack: 2.52 Kb
[528548537][lunokIoT] ESP32: Internal Free heap: 96.34 Kb
[528548937][lunokIoT] ESP32: Free heap: 3.60 MByte
[528559655][lunokIoT] ESP32: Heap hole: 935.28 Kb
[528560594][lunokIoT] ESP32: Most allocable data: 2.69 MByte (fragmentation: 25.4%)
[528572083][lunokIoT] ESP32: Heap integrity: good
[528572547][lunokIoT] ESP32: Tasks: 15
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x4015bed2  PS      : 0x00060830  A0      : 0x8015bfd0  A1      : 0x3ffe9e70  
A2      : 0x3ffdc22c  A3      : 0x3ffdbff4  A4      : 0x00000004  A5      : 0x00000001  
A6      : 0x3ffdc1e8  A7      : 0x003fffff  A8      : 0x00000001  A9      : 0x00000004  
A10     : 0x01ff0004  A11     : 0x003fffff  A12     : 0x80084ccf  A13     : 0x3ffe9db0  
A14     : 0x007ccee8  A15     : 0x003fffff  SAR     : 0x00000000  EXCCAUSE: 0x0000001c  
EXCVADDR: 0x01ff0004  LBEG    : 0x40092fe4  LEND    : 0x40092ffa  LCOUNT  : 0x00000000  


Backtrace:0x4015becf:0x3ffe9e700x4015bfcd:0x3ffe9ed0 0x400ef4f1:0x3ffe9ef0 0x400ecd83:0x3ffe9f20 0x400ecd97:0x3ffe9f40 0x402b92ff:0x3ffe9f60 0x402b937c:0x3ffe9fb0 
  #0  0x4015becf:0x3ffe9e700 in NimBLECharacteristic::notify(unsigned char const*, unsigned int, bool) at /home/avp/pio/ttgo/watch/.pio/libdeps/Debug-twatch2020_V3/NimBLE-Arduino/src/NimBLECharacteristic.cpp:449




ELF file SHA256: 0000000000000000

Rebooting...
[1080043][lunokIoT] 'компаньон' #24651//S9leI33k5B3lDvkjbY86QZLjS1GYvbyT//
```

**Comment by lunokjod** [comment]
seems be BLE issue

**Comment by lunokjod** [comment]
Solved on build #24714
